src/app/owner/page.tsx

```diff
@@ -0,0 +1,73 @@
+import Image, { StaticImageData } from 'next/image';
+import about from "@images/about/about1.png";
+
+
+const About = () => {
+  return (
+    <div className="min-h-screen bg-white">
+      <main className="max-w-4xl mx-auto p-4">
+        <nav className="text-sm mb-4 mt-[70px]">
+                  <a href="/" className="font-semibold text-sm leading-6 underline">powrót</a>
+        </nav>
+
+              <h3 className="font-bold text-sm leading-6 mt-[29px]">O nas // Katarzyna Bober </h3>
+        <h1 className="text-center font-dm-sans font-medium text-[44px] leading-[52.8px] tracking-[-0.04em] mt-[40px] ">Katarzyna Bober</h1>
+        <div className="flex justify-center mb-6">
+        </div>
+
+        <div>
+                  <Image src={about as StaticImageData}
+                      alt="Katarzyna Bober with dog"
+                      width={1217}
+                      height={600}
+            className="rounded-lg mt-[100px]" />
+           </div>
+
+          <div className="font-roboto font-normal text-sm leading-6 text-left mb-[80px] mt-[100px] ">
+            <h2 className='text-center font-dm-sans font-bold text-2xl leading-9 tracking-tight mb-[40px]'>O mnie</h2>
+            <h3 className=' font-bold text-base leading-[27px] mb-[16px]'>Wykształcenie i doświadczenie zawodowe</h3>
+            <ul className="list-disc">
+              <li>Technik weterynarii: Z zawodu jestem technikiem weterynarii. Po skończeniu szkoły odbyłam roczny staż w Opolskim Schronisku dla Bezdomnych Zwierząt, gdzie zdobyłam wiedzę o psim umyśle i ich potrzebach.</li>
+              <li>Zootechnika: Studiowałam zootechnikę we Wrocławiu.</li>
+              <li>Dyplom groomera: Dyplom groomera otrzymałam w szkole groomingu Pani Ewy Grabowskiej.</li>
+              <li>Szkolenie z medycyny behawioralnej: Uczęszczałam na szkolenie z zakresu medycyny behawioralnej psów i kotów.</li>
+            </ul>
+
+            <h3 className='font-bold text-base leading-[27px] mt-[26px] mb-[16px]'>Certyfikaty i osiągnięcia</h3>
+            <ul className="list-disc">
```

Comment:
For lists too, it is best to create another component with the example name ‘AboutList.tsx’ code example:
```

import React from 'react';

interface AboutListProps {
  items: string[];
}

const AboutList: React.FC<AboutListProps> = ({ items }) => {
  return (
    <ul className="list-disc pl-5">
      {items.map((item, index) => (
        <li key={index} className="text-lg leading-7 mb-2">
          {item}
        </li>
      ))}
    </ul>
  );
};

export default AboutList;
```

Reply:
In addition, ‘items’ can be a file with an array of descriptions. You add the file in AboutSection.tsx and pass it as props to AboutListItems.tsx where it will map all items :)

Reply:
done :)